website/content/en/docs/building-operators/golang/quickstart.md

```diff
@@ -1,500 +1,102 @@
 ---
-title: Golang Based Operator Quickstart
-linkTitle: Quickstart
-weight: 10
+title: Quickstart for Go-based Operators
+linkTitle: QuickStart
+weight: 20
+description: A simple set of instructions that demonstrates the basics of setting up and running a Go-based operator.
 ---
 
-**NOTE:** For the SDK versions prior to `v0.19.0` please consult the [legacy docs][legacy_quickstart_doc] for the [legacy CLI][legacy_CLI] and project.
-
-This guide walks through an example of building a simple memcached-operator using the operator-sdk CLI tool and controller-runtime library API.
```

Comment:
I don't think we need to change this line. It's good as is, IMO.

Reply:
- The note about v0.19 was removed since we have the docs released by version so it is not required at all
- The  second line still but in the description, as we did for helm`description: This guide walks through an example of building a simple memcached-operator using the operator-sdk CLI tool and controller-runtime library API.`

Reply:
@joelanford we have in the new quick:

````
---
title: Quickstart for Go-based Operators
linkTitle: QuickStart
weight: 20
description: A simple set of instructions that demonstrates the basics of setting up and running a Go-based operator.
---

This guide walks through an example of building a simple memcached-operator powered by Go using tools and libraries provided by the Operator SDK
```

Then see, the next steps: 

```
## Next Steps
Read the [tutorial][tutorial] for an in-depth walkthough of building a Go operator.
```

So, would not make sense keep the same. 


In this way for the tutorial we have: 

```
---
title: Golang Based Operator Tutorial
linkTitle: Tutorial
weight: 30
description: This guide walks through an example of building a simple memcached-operator using the operator-sdk CLI tool and controller-runtime library API.
---
```

Then, we just remove the note ` prior to `v0.19.0` because the docs are tagged already. So, would make sense keep it?